docs/sources/v0.50.x/javascript-api/k6-experimental/browser/page/throttlenetwork.md

```diff
@@ -11,8 +11,8 @@ Throttles the network in Chrome/Chromium to slow it down by the specified fields
 | ----------------------- | -------------- | ------- | -------------------------------------------------------------------------------------- |
 | networkProfile          | NetworkProfile | `null`  | This is a mandatory parameter.                                                         |
 | networkProfile.latency  | number         | `0`     | Minimum latency from request sent to response headers received (ms).                   |
-| networkProfile.download | number         | `-1`    | Maximal aggregated download throughput (bytes/sec). `-1` disables download throttling. |
-| networkProfile.upload   | number         | `-1`    | Maximal aggregated upload throughput (bytes/sec). `-1` disables upload throttling.     |
+| networkProfile.downloadThroughput | number         | `-1`    | Maximal aggregated download throughput (bytes/sec). `-1` disables download throttling. |
+| networkProfile.uploadThroughput   | number         | `-1`    | Maximal aggregated upload throughput (bytes/sec). `-1` disables upload throttling.     |
```

Comment:
Thanks for the change. The correct values should be `download` and `upload`. What's the reason for requesting the documentation change?

Reply:
Hi @ankur22,
I was able to use both download and downloadThroughput values. Some of my older scripts which uses 'download' were failing for some reason. But now I see both are working while I did the testing. So if there are notes saying 'downloadThroughput' is deprecated or shouldn't be used it would be great!

Reply:
There was a brief moment where `downloadThroughput` and `uploadThroughput` were part of a PR when the API was being built, but they were never released as supported fields for this API. I would recommend changing your scripts to work with `download` and `upload`.